core/src/main/scala/org/apache/spark/ui/jobs/StagePage.scala

```diff
@@ -177,6 +192,10 @@ private[ui] class StagePage(parent: StagesTab) extends WebUIPage("stage") {
           <div class="additional-metrics collapsed">
             <ul>
               <li>
+                <strong>Locality Level Summary: </strong>
```

Comment:
Yea, it didn't register to me that this was adding to the list of additional metrics enabled by checkboxes. If it goes here, I think it would have to be treated the same way. If a table displays well, that's reasonable to me. It might be harder to make render though; could be overkill at this stage. @kayousterhout has better sensibilities here.